src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -78,34 +93,166 @@ public void robotInit() {
     zeroElevator.setRunWhenDisabled(true);
     SmartDashboard.putData(zeroElevator);
 
-    UsbCamera camera = CameraServer.getInstance().startAutomaticCapture(Tuning.camID);
-    camera.setResolution(128, 73);
-    camera.setFPS(30);
+    Command zeroDrivetrain = new SimpleCommand("[Drivetrain] Zero Drivetrain", drivetrain::zeroEncoders);
+    zeroDrivetrain.setRunWhenDisabled(true);
+    SmartDashboard.putData(zeroDrivetrain);
```

Comment:
Can we centralize all the SmartDashboard putting? It's awfully annoying to have it spread around.

Reply:
It's all in `robotInit()` for stuff that's set-and-forget and `robotPeriodic()` for telemetry, don't know how more centralized you want to be

Reply:
Putting it in the same parts of said methods.